### ISSUE TITLE
API doesn't return full text of long tweets

### ISSUE THREAD

**Issue by MFGood** [FxEmbed/FxEmbed#673]
On tweets where you have to click "Show more" to view the whole text, the API only returns the truncated text. However, if you translate that same tweet to a different language, you get the full thing translated. I could see how the truncated text might be desirable to some developers, but could there possibly be another property added to the response called something like text_full? I know you're working on a new version of the API, and maybe you've already got a feature like this in the works, but thought I'd mention it.

**Comment by Polsy** [comment]
Is this not already fixed? I've been using the fxtwitter api precisely because it does return the full text of long tweets (unlike vx) and I've never used the translation function.

**Comment by dangeredwolf** [comment]
@Polsy Yes I think it is, sorry I'm going through stale issues and I noticed this
